@stellar/anchor-tests/src/helpers/sep24.ts

```diff
@@ -0,0 +1,68 @@
+import { Request } from "node-fetch";
+
+import { Result, NetworkCall, Failure } from "../types";
+import { makeRequest } from "./request";
+
+export const invalidTransactionSchema: Failure = {
+  name: "invalid transaction schema",
+  text(args: any): string {
+    return (
+      "The response body returned does not comply with the schema defined for the /transaction endpoint. " +
+      "The errors returned from the schema validation:\n\n" +
+      `${args.errors}.`
+    );
+  },
+  links: {
+    "Transaction Schema":
+      "https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0024.md#single-historical-transaction",
+  },
+};
+
+export const unexpectedTransactionStatus: Failure = {
+  name: "unexpected transaction status",
+  text(args: any): string {
+    return `Unexpected transaction status. Expected '${args.expected}' but received '${args.received}' instead.`;
+  },
+};
+
+export const invalidConfigFile: Failure = {
+  name: "invalid config file",
+  text(args: any): string {
+    return "The UPLOAD CONFIG file has some issues:\n\n" + `${args.errors}.`;
```

Comment:
Can we update this message to account for the possibility that the config file wasn't provided?

Reply:
I'll update the message

Reply:
Just pushed this change, this is how it'll look like when the Sep-24 config object is missing:
![Screenshot 2023-07-31 at 16 24 35](https://github.com/stellar/stellar-anchor-tests/assets/3228151/3c0b9b62-ac12-4d41-831a-c68709355acf)